mybatis-spring-boot-samples/mybatis-spring-boot-sample-xml/src/main/java/sample/mybatis/xml/mapper/CityMapper.java

```diff
@@ -16,8 +16,6 @@
 package sample.mybatis.xml.mapper;
 
 import org.apache.ibatis.annotations.Mapper;
```

Comment:
```suggestion
import org.apache.ibatis.annotations.Mapper;
import org.apache.ibatis.annotations.Param;
```

Reply:
@awxiaoxian2020 Thanks for your reaction!
@imba-tjd please fix build error...